assets/minecraft/lang/zh_cn.json

```diff
@@ -0,0 +1,198 @@
+  "menu.nova.recipe": "合成指南",
+  "menu.nova.recipe.back": "返回",
+  "menu.nova.recipe.pulverize": "粉碎",
+  "menu.nova.recipe.plate": "冲压（板）",
+  "menu.nova.recipe.gear": "冲压（齿轮）",
+  "menu.nova.side_config": "配置",
+  "menu.nova.side_config.energy": "电力配置",
+  "menu.nova.side_config.item": "物品配置",
+  "menu.nova.side_config.front": "§7前",
+  "menu.nova.side_config.left": "§7左",
+  "menu.nova.side_config.back": "§7后",
+  "menu.nova.side_config.right": "§7右",
+  "menu.nova.side_config.top": "§7上",
+  "menu.nova.side_config.bottom": "§7下",
+  "menu.nova.side_config.none": "§7无",
+  "menu.nova.side_config.output": "§6输出",
+  "menu.nova.side_config.input": "§b输入",
+  "menu.nova.side_config.input_output": "§a输入 & 输出",
+  "menu.nova.cable_config.insert": "输入",
+  "menu.nova.cable_config.extract": "输出",
+  "menu.nova.paged.back": "§7返回",
+  "menu.nova.paged.forward": "§7下一页",
+  "menu.nova.paged.go_infinite": "§8前往第 %s 页",
+  "menu.nova.paged.go": "§8前往第 %s/%s 页",
+  "menu.nova.paged.limit_min": "§8你不能再返回了",
+  "menu.nova.paged.limit_max": "§8没有更多页面了",
+  "menu.nova.visual_region.hide": "隐藏区域",
+  "menu.nova.visual_region.show": "显示区域",
+  "menu.nova.storage_unit.item_display_plural": "%s §7物品",
+  "menu.nova.storage_unit.item_display_singular": "%s §7物品",
+  "menu.nova.block_breaker": "方块破坏机",
+  "menu.nova.block_placer": "方块放置机",
+  "menu.nova.breeder": "饲养机",
+  "menu.nova.breeder.idle": "下一次饲养将在 %s ticks 后",
+  "menu.nova.cable_config": "配置",
+  "menu.nova.charger": "充电器",
+  "menu.nova.chunk_loader": "区块加载器",
+  "menu.nova.electrical_furnace": "电炉",
+  "menu.nova.furnace_generator": "火力发电机",
+  "menu.nova.mechanical_press": "金属冲压机",
+  "menu.nova.mechanical_press.press_plates": "冲压（板）",
+  "menu.nova.mechanical_press.press_gears": "冲压（齿轮）",
+  "menu.nova.mob_duplicator": "生物克隆机",
+  "menu.nova.mob_duplicator.idle": "下一次生成将在 %s ticks 后",
+  "menu.nova.mob_duplicator.nbt": "使用 NBT-数据",
+  "menu.nova.mob_killer": "生物屠宰机",
+  "menu.nova.mob_killer.idle": "下一次攻击将在 %s ticks 后",
+  "menu.nova.power_cell": "蓄电池",
+  "menu.nova.pulverizer": "粉碎机",
+  "menu.nova.quarry": "采石场",
+  "menu.nova.quarry.size": "采石场开采面积: %s x %s 方块",
+  "menu.nova.quarry.size_tip": "开采面积越大，采石场的效率越低",
+  "menu.nova.solar_panel": "太阳能电池板",
+  "menu.nova.storage_unit": "存储单元",
+  "menu.nova.vacuum_chest": "虚空箱",
+  "menu.nova.item_filter": "物品过滤器",
+  "menu.nova.item_filter.whitelist": "当前过滤模式: 白名单",
+  "menu.nova.item_filter.blacklist": "当前过滤模式: 黑名单",
+
+  "command.nova.give.success": "成功给予 %s 到你的物品栏中.",
+  "command.nova.list_nearby.success": "在你所处区块的 %s 个盔甲架中, %s 个是 tile 实体.",
+  "command.nova.get_nearest_data.failed": "无法找到在附近找到 tile 实体.",
+  "command.nova.remove_obsolete_models.success": "成功移除 %s 个盔甲架.",
+  "command.nova.remove_tile_entities.success": "成功移除 %s 个 tile 实体.",
+  "command.nova.network_debug.energy": "切换 电力网络 调试视角",
+  "command.nova.network_debug.item": "切换 物品网络 调试视角",
+
+  "advancement.nova.root.title": "Nova",
+  "advancement.nova.root.description": "汉化:小默米",
```

Comment:
Credits here are fine for now. We will add an extra key for credits in the future